tests/foreman/cli/test_os.py

```diff
@@ -224,24 +165,18 @@ def test_positive_update_1(self, test_data):
         @assert: Operating System is updated and can be found
 
         """
-        # "Unpacks" values from tuple
-        orig_dict, updates_dict = test_data
-        # Create a new object passing @test_data to factory method
-        os = make_os(orig_dict)
-        # Update original test_data with new values
-        updates_dict['id'] = os['id']
-        orig_dict.update(updates_dict)
-        # Now update the Foreman object
```

Comment:
Comments are necessary but only if required.  You dont need to necessary comment any obvious line of code like this old test.  All these unnecessary comments are removed in this PR.